learning_resources/views.py

```diff
@@ -845,6 +845,6 @@ def get_queryset(self) -> QuerySet:
             .filter(parents__parent_id__in=featured_list_ids)
             .filter(published=True)
             .annotate(position=F("parents__position"))
-            .order_by("position")
+            .order_by("position", "?")
```

Comment:
A potential issue is that using `?` with `order_by` interferes with pagination: `/api/v1/featured/?limit=10&offset=0` and `/api/v1/featured/?limit=10&offset=10` can now overlap. The frontend is currently only requesting a single page of results, so maybe this isn't a realistic issue.

Could the randomizing happen in Python after fetching from db?

Reply:
Refactored it to do the randomizing in python